projects/ad405x_iio/app/ad405x_iio.c

```diff
@@ -1492,6 +1499,10 @@ int32_t iio_ad405x_initialize(void)
 		}
 	};
 
+	no_os_udelay(
```

Comment:
curious to know why we need such a huge delay in two instances? 
Did you face any issues without it?

Reply:
Yes, in fact we will face it with all of our projects on reconnection of SDP-K1 with the PC. The context attributes don't load unless doing a reset on the board (or reprogramming the board) without this fix..

This is more evident in projects that require external power supply (along with the SDP-K1 connection)... around 3-4 secs as mentioned by Himanshu!

Reply:
Got it. 
Have we mentioned this to the ACE team? The ACE plugin sort of flashes the IIO firmware during start up. It would be good to check up if there aren't any timeout restrictions on the ACE front

Reply:
Yes, I got this checked and tested with ACE plug-in (with Cathal) for Blackwater.